src/activities/guesscount-summer/GuesscountSummer.qml

```diff
@@ -0,0 +1,374 @@
+            onClose: home()
+        }
+
+        DialogActivityConfig {
+            id: dialogActivityConfig
+            currentActivity: activity
+            content: Component {
+                Item {
+                    property alias modeBox: modeBox
+                    property var availableModes: [
+                        { "text": qsTr("Admin"), "value": "admin" },
+                        { "text": qsTr("BuiltIn"), "value": "builtin" }
+                    ]
+
+                    Rectangle {
+                        id: flow
+                        width: dialogActivityConfig.width
+                        height: background.height
+                        GCComboBox {
+                            id: modeBox
+                            anchors {
+                                top: parent.top
+                                topMargin: 5
+                            }
+
+                            model: availableModes
+                            background: dialogActivityConfig
+                            label: qsTr("Select your mode")
+                        }
+                        Row {
+                            id: labels
+                            spacing: 20
+                            anchors {
+                                top: modeBox.bottom
+                                topMargin: 5
+                            }
+                            visible: modeBox.currentIndex == 0
+                            Repeater {
+                                model: 2
+                                Row {
+                                    spacing: 10
+                                    Rectangle {
+                                        id: label
+                                        width: background.width*0.3
+                                        height: background.height/15
+                                        radius: 20.0;
+                                        color: modelData ? "green" : "red"
+                                        GCText {
+                                            anchors.horizontalCenter: parent.horizontalCenter
+                                            anchors.verticalCenter: parent.verticalCenter
+                                            fontSize: smallSize
+                                            text: modelData ? qsTr("Selected") : qsTr("Not Selected")
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                        Rectangle {
+                            width: parent.width
+                            color: "transparent"
+                            height: parent.height/1.25-labels.height-modeBox.height
+                            anchors {
+                                top: labels.bottom
+                                topMargin: 5
+                            }
+                            ListView {
```

Comment:
In landscape mode (on phone and tablet), I still don't see entirely the last line.